test/pet.test.js

```diff
@@ -1,24 +1,246 @@
 const Pet = require('../src/pet');
 
-describe('constructor', () => {
-  let pet;
+
+  let petCat;
 
   beforeEach(() => {
-     pet = new Pet('Rocky');
-     expect(new Pet('Rocky')).toBeInstanceOf(Object);
-     expect(pet.name).toEqual('Rocky');
+     petCat = new Pet('Rocky');    
 
   });
 
-  it('has a initial age of 0 & hunger of 0 & fitness of 10', () => {
+describe('constructor', () => {
 
-      expect(pet.age).toEqual(0);
-      expect(pet.hunger).toEqual(0);
-      expect(pet.fitness).toEqual(10);
-       
-    });
+  it ('Creates petCat as an instance of an Object', () => {
+    expect(new Pet('Rocky')).toBeInstanceOf(Object);
+
+  });
+
+  it ('petCat has the specified name', () => {
+
+    petCat.name = 'Rocky';
+    expect(petCat.name).toEqual('Rocky');
+  });
+
+
+  it('petCat has a initial age of 0', () => {
+
+    expect(petCat.age).toEqual(0);
+   
+  });
+
+  it('has a initial hunger of 0', () => {
+
+    expect(petCat.hunger).toEqual(0);
+  });
+
+  it('has a initial fitness of 0', () => {
+
+    expect(petCat.fitness).toEqual(10);
+  });
+  
+}); 
+
+describe('growUp', () => {
+
+  it('increments petCat\'s age by 1', () => {
+    //petCat.age = 0;
+    petCat.growUp();
+    expect (petCat.age).toEqual(1);
+
+    petCat.growUp();
+    expect (petCat.age).toEqual(2);
+
+  });
+
+  it ('increments hunger by 5', () => {
+    petCat.growUp();
+    expect(petCat.hunger).toEqual(5);
+
+    petCat.hunger = 3;    
+    petCat.growUp();
```

Comment:
Really good you're testing it multiple times in the same block. Means you know it doesn't just work for one case.